app/code/core/Mage/Admin/Model/User.php

```diff
@@ -733,17 +733,21 @@ protected function _getDateNow($dayOnly = false)
      * You can declare additional emails in Mage_Core general/additional_notification_emails/admin_user_create node.
      *
      * @param Mage_Admin_Model_User $user
+     * @param string|false $receiverEmail An optional receiver email address for the notification. Overrides default and additional emails.
      * @return $this
      */
-    public function sendAdminNotification($user)
+    public function sendAdminNotification($user, $receiverEmail = false)
     {
         // define general contact Name and Email
         $generalContactName = Mage::getStoreConfig('trans_email/ident_general/name');
         $generalContactEmail = Mage::getStoreConfig('trans_email/ident_general/email');
 
-        // collect general and additional emails
-        $emails = $this->getUserCreateAdditionalEmail();
-        $emails[] = $generalContactEmail;
+        // collect general and additional emails unless overridden
+        if ($receiverEmail && filter_var($receiverEmail, FILTER_VALIDATE_EMAIL)) {
```

Comment:
`Zend_Validate::is($receiverEmail, 'EmailAddress')` perhaps?

Reply:
Yes, because `FILTER_VALIDATE_EMAIL` is not used.